ref/ams-configuration.yaml

```diff
@@ -176,53 +244,61 @@ all:
       [Anbox Application Registry](https://discourse.ubuntu.com/t/application-registry/17761)
       uses to TLS-secure its HTTPS endpoint. This is used by AMS for
       mutual TLS authentication with the registry.
+    deprecated: false
   registry.mode:
     type: "string"
     default: "`pull`"
     description: >-
       Mode in which the
       [Anbox Application Registry](https://discourse.ubuntu.com/t/application-registry/17761)
       client in AMS operates: `manual`, `pull`, `push`
+    deprecated: false
   registry.update_interval:
     type: "string"
     default: "`1h`"
     description: >-
       Frequency of
       [Anbox Application Registry](https://discourse.ubuntu.com/t/application-registry/17761)
       updates (for example: 1h, 30m).
+    deprecated: false
```

Comment:
We do not necessarily have to put the `false` value as it it considered the default for a boolean but also not a problem, so lets keep as is.